### PR TITLE
Add initial GitLabChangelog package files

### PR DESCRIPTION
**Pull request by nicoleepp**
Part of https://github.com/invenia/TagBotGitLab/issues/22

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,24 +7,32 @@
...
 EXTRAS = {
     "test": TEST_DEPS,
     "docs": DOCS_DEPS,
     "check": CHECK_DEPS,
     "dev": TEST_DEPS + DOCS_DEPS + CHECK_DEPS,
 }
 
+
+REQUIREMENTS = [
+    "invenia-cli-utils",
```
Comment (nicoleepp) [inline setup.py:19]: Why is this required?

Comment (fchorney) [reply setup.py:19]: Looks like this is a bug in our template creator. This isn't needed for github projects

File: pyproject.toml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 [tool.black]
-target-version = ["py37", "py38"]
+target-version = ["py38"]
```
Comment (fchorney) [inline pyproject.toml:2]: You could remove the `.coveragerc` file and put it's contents in here like so

```toml

[tool.coverage.run]
relative_files = true

[tool.coverage.report]
precision = 2
show_missing = true
skip_covered = true
exclude_lines = ["if __name__ == .__main__.:", "def __str__", "def __repr__", "pragma: no cover"]
```

you can also put isort contents in here as well, but it doesn't look like flake8 supports project.toml. Putting the converage config in here at least lets you drop the other config file.

File: LICENSE [hunk 3]
```diff
@@ -1,3 +1,5 @@
+MIT License
```
Comment (fchorney) [inline LICENSE:1]: Do we need this line? Github seems to be able to figure out the licence without it.

File: pyproject.toml (hunk 2 above)
Comment (nicoleepp) [reply pyproject.toml:2]: I put isort and flake8 into the setup.cfg file

File: LICENSE (hunk 3 above)
Comment (nicoleepp) [reply LICENSE:1]: From the [Github docs](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository#determining-the-location-of-your-license):


> Most people place their license text in a file named LICENSE.txt (or LICENSE.md) in the root of the repository; here's an example from Hubot.
> 
> Some projects include information about their license in their README. For example, a project's README may include a note saying "This project is licensed under the terms of the MIT license."
> 
> As a best practice, we encourage you to include the license file with your project.

Comment (fchorney) [reply LICENSE:1]: I don't feel like that quote references having this one line in this license file?

File: pyproject.toml (hunk 2 above)
Comment (fchorney) [reply pyproject.toml:2]: yeah I think it's fine to put some in setup.cfg and some in pyproject.toml. It's annoying to me that all these tools haven't adopted pyproject.toml though.

File: setup.py [hunk 4]
```diff
@@ -7,24 +7,32 @@
 TEST_DEPS = ["coverage", "pytest", "pytest-cov"]
```
Comment (fchorney) [inline setup.py:7]:
```suggestion
TEST_DEPS = ["coverage[toml]", "pytest", "pytest-cov"]
```

File: tox.ini [hunk 5]
```diff
@@ -16,7 +16,7 @@ commands =
     py.test --cov=gitlabchangelog --verbose --tb=long
 
 [testenv:coverage]
-basepython = python3.7
+basepython = python3
 deps = coverage
```
Comment (fchorney) [inline tox.ini:20]:
```suggestion
deps = coverage[toml]
```

File: LICENSE (hunk 3 above)
Comment (nicoleepp) [reply LICENSE:1]: Oh, I read "Do we need this file", no we probably don't need it but I like it for clarity of the reader and have seen other projects include it

Comment (fchorney) [reply LICENSE:1]: ahh ok. That should be fine then, I haven't seen it myself so I wasn't sure.

**Review by fchorney — APPROVED** [review]
